pkgs/development/compilers/lessc/default.nix

```diff
@@ -23,7 +23,7 @@ stdenv.mkDerivation rec {
     description = "LESS to CSS compiler";
     homepage = http://lesscss.org/;
     license = licenses.asl20;
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
Now that this has been merged I should be able to nix-channel --update and use lessc on a darwin system correct? Because I'm still getting a not supported platform error.